recipes/h5glance/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "h5glance" %}
+{% set org = "European-XFEL" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: dd848a967112f0a0d1d04a1739afbe7dc7bac682b93b8017e5d5a11ca8260c5d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - h5glance = {{ name }}.terminal:main
+    - h5glance-html = {{ name }}.html_cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >=3.7
+    - h5py >=2.10
+    - htmlgen
+
+test:
+  requires:
+    - pip
+  imports:
+    - {{ name }}
+  commands:
+    - pip check
+    - h5glance --version | grep {{ version }}
+    - h5glance-html -h
+
+about:
+  home: https://github.com/{{ org }}/{{ name }}
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: Lets you explore HDF5 files in the terminal or an HTML interface.
+  description: Lets you explore HDF5 files in the terminal or an HTML interface.
+  doc_url: https://github.com/{{ org }}/{{ name }}
+  dev_url: https://github.com/{{ org }}/{{ name }}
```

Comment:
I'd rather not use Jinja here so humans can copy paste URLs from the recipe directly. Makes it more difficult to parse. It's not like that org or repo names will change often anyway.

Reply:
Fine. I developed this habit to avoid copy/past mistakes for new feedstock, but I will change